flink-core/src/main/java/org/apache/flink/configuration/CoreOptions.java

```diff
@@ -140,6 +140,10 @@
 		.key("env.java.opts.taskmanager")
 		.defaultValue("");
 
+	public static final ConfigOption<String> FLINK_HS_JVM_OPTIONS = ConfigOptions
+		.key("env.java.opts.historyserver")
+		.defaultValue("");
```

Comment:
Would be good to add a description.

Reply:
@tillrohrmann Thanks for your review.  As for the description， I think the annotation  "process parameters" in line 128 of this class already has described “FLINK_JVM_OPTIONS”，“FLINK_JM_JVM_OPTIONS”，“FLINK_TM_JVM_OPTIONS”，as well as our “FLINK_HS_JVM_OPTIONS”.